### PR TITLE
Maintenance/ember 2.15

### PR DESCRIPTION
**Pull request by ultimatemonty**
Upgrades us to all things 2.15 (ember-cli, ember, ember-data).

I use version diffs from the [ember-cli/ember-new-output](https://github.com/ember-cli/ember-new-output/compare/v2.12.0...v2.15.0) repo for upgrades

### CODE REVIEW

File: testem.js [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+  disable_watching: true,
+  launch_in_ci: [
+    'Chrome'
   ],
-  "launch_in_dev": [
-    "PhantomJS",
-    "Chrome"
-  ]
+  launch_in_dev: [
+    'Chrome'
```
Comment (ultimatemonty) [inline testem.js:9]: no more Phantom - 2.15 defaults to headless Chrome for all tests. See the [release blog post](https://emberjs.com/blog/2017/09/01/ember-2-15-released.html#toc_changes-in-ember-cli-2-15) for deets

File: app/app.js [hunk 2]
```diff
@@ -3,11 +3,7 @@ import Resolver from './resolver';
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
-let App;
-
-Ember.MODEL_FACTORY_INJECTIONS = true;
```
Comment (gschorkopf) [inline app/app.js:8]: Why does this go away?

File: app/index.html [hunk 3]
```diff
@@ -9,8 +9,8 @@
 
     {{content-for "head"}}
 
-    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/frolfr-web.css">
+    <link integrity="" rel="stylesheet" href="{{rootURL}}assets/vendor.css">
```
Comment (gschorkopf) [inline app/index.html:12]: What's this?

File: config/environment.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 /* eslint-env node */
 
 module.exports = function(environment) {
-  var ENV = {
+  let ENV = {
```
Comment (gschorkopf) [inline config/environment.js:4]: 👍

File: config/environment.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
     apiHost: '',
     modulePrefix: 'frolfr-web',
-    environment: environment,
+    environment,
```
Comment (gschorkopf) [inline config/environment.js:7]: 👍

**Review by gschorkopf — APPROVED** [review]
I'd love to get some feedback on the couple of questions I asked. Otherwise this looks bueno! I guess I should probably look into upgrading at work too... 😄

File: app/index.html (hunk 3 above)
Comment (ultimatemonty) [reply app/index.html:12]: I believe this is needed for the ember-sri addon (SubResource Integrity). See https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

File: app/app.js (hunk 2 above)
Comment (ultimatemonty) [reply app/app.js:8]: looks like it's no longer needed. https://github.com/emberjs/ember.js/pull/15204

**Comment by gschorkopf** [comment]
> I use version diffs from the ember-cli/ember-new-output repo for upgrades

Did you do that manually, or use `ember init`?

**Comment by ultimatemonty** [comment]
I manually do it. `ember init` tends to wipe out some changes. Some folks will `ember init` and then look at the diffs for each file that changed. I prefer to just do it manually via the diffs from the ember-new-output project.

**Pull request merged**